website/content/docs/agent/autoauth/index.mdx

```diff
@@ -138,8 +138,13 @@ These are common configuration values that live within the `method` block:
   structure. Values can be an integer number of seconds or a stringish value
   like `5m`.
 
+- `min_backoff` `(string or integer: "1s")` - The initial backoff time Agent 
```

Comment:
Should be parameter be `min_backoff` or `initial_backoff`? I see mixed references between min and initial even though they refer to the same thing.